packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -41,11 +41,11 @@ import type {
   StringElement,
 } from './component';
 import type {
-  AddressInputElement,
   AssetSelectorElement,
   AvatarElement,
   SkeletonElement,
   AddressElement,
+  AddressInputElement,
```

Comment:
Can we revert this unnecessary change?

Reply:
?

Reply:
oh it was changed to be sorted, I guess that's fine